src/components/Rating/Rating.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { Checkbox as CoreCheckbox } from 'wix-ui-core/checkbox';
+import styles from './Rating.st.css';
+import { ReactComponent as StarIcon } from '../../assets/icons/Star.svg';
+
+export enum IconSize {
+  LARGE = 'large',
+}
+
+export enum Mode {
+  EDIT = 'edit',
+  PREVIEW = 'preview',
+}
+
+export interface RatingProps {
+  value: number;
+  mode: Mode;
+  onSelect?(value: number): void;
+  maxRating?: number;
```

Comment:
I talked to the designers, and they said they want the rating component to have only 5 stars.
So I guess we should remove the `maxRating` prop.